bluepysnap/utils.py

```diff
@@ -58,16 +57,11 @@ def roundrobin(*iterables):
     num_active = len(iterables)
 
     # cannot use six.next. Need the function and not the call and cannot use ternary operator
-    if six.PY3:  # pragma: no cover
```

Comment:
https://more-itertools.readthedocs.io/en/stable/api.html#more_itertools.roundrobin

But I know you don't like it 
It's awesome though !!!

Reply:
yeah you convince me with the no dependency dragged argument.